ionc/ion_writer_text.c

```diff
@@ -667,7 +730,11 @@ iERR _ion_writer_text_write_timestamp(ION_WRITER *pwriter, iTIMESTAMP value)
         temp[output_length] = '\0';
 
         // and our helper does the rest
+        if (json_downconvert)
+           ION_PUT(pwriter->output, '"');
         IONCHECK(_ion_writer_text_append_ascii_cstr(pwriter->output, temp));
+        if (json_downconvert)
+           ION_PUT(pwriter->output, '"');
```

Comment:
We haven't defined a formal style guide for this repo, but let's use braces for every `if` (for all new code anyway).